### ISSUE TITLE
[mainnet] Rewards differ between ledger and db-sync

### ISSUE THREAD

**Issue by yorickvP** [IntersectMBO/cardano-db-sync#939]
**OS**
Your OS: NixOS

**Versions**
The `db-sync` version (eg `cardano-db-sync --version`):
```
cardano-db-sync 11.0.4 - linux-x86_64 - ghc-8.10
git revision 1e964e0cc35a5466903aada0c6b29b71d6b82a28
```
PostgreSQL version: 10.18

**Build/Install Method**
The method you use to build or install `cardano-db-sync`:
the exact instructions from https://github.com/input-output-hk/cardano-db-sync/blob/master/doc/building-running.md

**Run method**
The method you used to run `cardano-db-sync` (eg Nix/Docker/systemd/none):
Nix

**Additional context**

**Problem Report**
Please do not include screenshots or images, but instead cut and paste any relevant log messages
or errors.
```
ESC[34m[db-sync-node:Info:6]ESC[0m [2021-11-08 09:24:20.86 UTC] Schema migration files validated
ESC[34m[db-sync-node:Info:6]ESC[0m [2021-11-08 09:24:20.86 UTC] Running database migrations
ESC[34m[db-sync-node:Info:6]ESC[0m [2021-11-08 09:24:21.00 UTC] Using byron genesis file from: "config/../../cardano-node/configuration/cardano/mainnet-byron-genesis.json"
ESC[34m[db-sync-node:Info:6]ESC[0m [2021-11-08 09:24:21.00 UTC] Using shelley genesis file from: "config/../../cardano-node/configuration/cardano/mainnet-shelley-genesis.json"
ESC[34m[db-sync-node:Info:6]ESC[0m [2021-11-08 09:24:21.00 UTC] Using alonzo genesis file from: "config/../../cardano-node/configuration/cardano/mainnet-alonzo-genesis.json"
ESC[34m[db-sync-node:Info:6]ESC[0m [2021-11-08 09:24:21.25 UTC] NetworkMagic: 764824073
ESC[34m[db-sync-node:Info:6]ESC[0m [2021-11-08 09:24:21.48 UTC] Initial genesis distribution present and correct
ESC[34m[db-sync-node:Info:6]ESC[0m [2021-11-08 09:24:21.48 UTC] Total genesis supply of Ada: 31112484745.000000
ESC[34m[db-sync-node:Info:6]ESC[0m [2021-11-08 09:24:21.48 UTC] Inserting Shelley Genesis distribution
ESC[34m[db-sync-node:Info:6]ESC[0m [2021-11-08 09:24:21.48 UTC] localInitiatorNetworkApplication: connecting to node via "../cardano-node/state-node-mainnet/node.socket"
ESC[35m[db-sync-node.Subscription:Notice:49]ESC[0m [2021-11-08 09:24:21.48 UTC] Identity Starting Subscription Worker, valency 1
ESC[35m[db-sync-node.Subscription:Notice:50]ESC[0m [2021-11-08 09:24:21.48 UTC] Identity Connection Attempt Start, destination LocalAddress "../cardano-node/state-node-mainnet/node.socket"
ESC[35m[db-sync-node.Subscription:Notice:50]ESC[0m [2021-11-08 09:24:21.48 UTC] Identity Connection Attempt End, destination LocalAddress "../cardano-node/state-node-mainnet/node.socket" outcome: ConnectSuccessLast
ESC[34m[db-sync-node.Handshake:Info:50]ESC[0m [2021-11-08 09:24:21.48 UTC] WithMuxBearer (ConnectionId {localAddress = LocalAddress "", remoteAddress = LocalAddress "state-node-mainnet/node.socket"}) Send (ClientAgency TokPropose,MsgProposeVersions (fromL
ist [(NodeToClientV_1,TInt 764824073),(NodeToClientV_2,TInt 764824073),(NodeToClientV_3,TInt 764824073),(NodeToClientV_4,TInt 764824073),(NodeToClientV_5,TInt 764824073),(NodeToClientV_6,TInt 764824073),(NodeToClientV_7,TInt 764824073),(NodeToClientV_8,TI
nt 764824073),(NodeToClientV_9,TInt 764824073),(NodeToClientV_10,TInt 764824073)]))
ESC[34m[db-sync-node.Handshake:Info:50]ESC[0m [2021-11-08 09:24:21.48 UTC] WithMuxBearer (ConnectionId {localAddress = LocalAddress "", remoteAddress = LocalAddress "state-node-mainnet/node.socket"}) Recv (ServerAgency TokConfirm,MsgAcceptVersion NodeToCl
ientV_10 (TInt 764824073))
ESC[34m[db-sync-node.Mux:Info:50]ESC[0m [2021-11-08 09:24:21.48 UTC] Bearer on ConnectionId {localAddress = LocalAddress "", remoteAddress = LocalAddress "state-node-mainnet/node.socket"} event: Handshake Client end, duration 0.00020135s
ESC[34m[db-sync-node.Mux:Info:51]ESC[0m [2021-11-08 09:24:21.48 UTC] Bearer on ConnectionId {localAddress = LocalAddress "", remoteAddress = LocalAddress "state-node-mainnet/node.socket"} event: State: Mature
ESC[34m[db-sync-node:Info:54]ESC[0m [2021-11-08 09:24:21.48 UTC] Starting chainSyncClient
ESC[34m[db-sync-node:Info:54]ESC[0m [2021-11-08 09:24:21.49 UTC] Cardano.Db tip is at slot 44668784, block 6468252
ESC[34m[db-sync-node:Info:59]ESC[0m [2021-11-08 09:24:21.49 UTC] Running DB thread
ESC[34m[db-sync-node:Info:59]ESC[0m [2021-11-08 09:24:21.49 UTC] Rolling back to slot 44668784, hash 9992d96ae5507f4ed2386ce2e89d32e7fc8e2c1b098df73cf660a3cd743029fe
ESC[34m[db-sync-node:Info:59]ESC[0m [2021-11-08 09:24:21.49 UTC] No blocks need to be deleted
ESC[35m[db-sync-node.Subscription:Notice:49]ESC[0m [2021-11-08 09:24:21.71 UTC] Identity Required subscriptions started
ESC[34m[db-sync-node:Info:59]ESC[0m [2021-11-08 09:24:35.39 UTC] Found snapshot file for slot 44668784, hash 9992d96ae5507f4ed2386ce2e89d32e7fc8e2c1b098df73cf660a3cd743029fe
ESC[34m[db-sync-node:Info:62]ESC[0m [2021-11-08 09:24:41.29 UTC] getHistoryInterpreter: acquired
ESC[34m[db-sync-node:Info:72]ESC[0m [2021-11-08 09:24:41.29 UTC] Running Offline fetch thread
ESC[34m[db-sync-node:Info:59]ESC[0m [2021-11-08 09:24:41.29 UTC] thisIsAnUglyHack: Main thead
ESC[34m[db-sync-node:Info:59]ESC[0m [2021-11-08 09:24:59.73 UTC] File ledger-state/mainnet/44668784-9992d96ae5-300.lstate exists
ESC[34m[db-sync-node:Info:59]ESC[0m [2021-11-08 09:24:59.73 UTC] Starting at epoch 301
ESC[34m[db-sync-node:Info:59]ESC[0m [2021-11-08 09:25:00.71 UTC] Handling 928654 stakes for epoch 301 slot 44668896, hash 7f119b06918c6c9f6d63732d5a440e2bbb464e81cafb8d42318a03defc18fc4b
ESC[33m[db-sync-node:Warning:59]ESC[0m [2021-11-08 09:26:00.72 UTC] validateEpochRewards: rewards spendable in epoch 301 expected total of 14706860.245562 ADA but got 14968906.805906 ADA
Epoch No: 301
dbMap length: 860850
ledgerMap length: 860850
diffMap length: 43761
```
43.000 lines skipped
```
panic abort:
Rewards differ between ledger and db-sync.
Please report at https://github.com/input-output-hk/cardano-db-sync/issues.
```

**Comment by erikd** [comment]
This should be fixed fixed in the `12.0.0` release which should be released some time this week or early next week.

**Comment by Cmdv** [comment]
It looks like there hasn't been any activity on this issue for a some time. We're closing it for now. If you're still experiencing this problem or have any updates, please reopen or create a new issue. Your feedback is valuable, thank you!
